assets/js/remotebuzzer-server.js

```diff
@@ -232,6 +236,42 @@ const requestListener = function (req, res) {
                 'text/html'
             );
             break;
+        case '/commands/increase-print-limit':
+            log('http: GET /commands/increase-print-limit');
+            log('current directory', __dirname);
+            if (config.remotebuzzer.usebuttons) {
+                if (config.print.from_result || config.print.from_gallery) {
+                    let i = 1;
+                    let j = parseInt(queryParams.get('i'), 10);
+                    if (j) {
+                        i = j;
+                    }
+                    //Read config file and find the current print limit
+
+                    const configPath = path.join(__dirname, '..', '..', 'config', 'my.config.inc.php');
```

Comment:
Not needed with the new API endpoint. Only needs a get/post request with `i` to increase the amount.